README.md

```diff
@@ -102,14 +111,14 @@ JSON endpoints:
    "ram_mb":8192
 }
 ```
-- **`/benchmarks`**
+- **`/benchmarks`**: TODO: seems like this section is work-in-progress?
```

Comment:
Yes - I think it's fair to say that. Would benefit if we decide on some concrete parameters that could be consistent over time

Reply:
Ok, let's fix that in https://github.com/paritytech/substrate-analytics/issues/61